src/server.ts

```diff
@@ -68,12 +69,12 @@ async function validate(document: TextDocument) {
                 let newDocumentsWithErrors: any = [];
                 for (let fileName in compileErrorDiagnostics) {
                     newDocumentsWithErrors.push(URI.file(fileName).path);
-                    connection.sendDiagnostics({diagnostics: compileErrorDiagnostics[fileName], uri: URI.file(fileName).path});
+                    connection.sendDiagnostics({diagnostics: compileErrorDiagnostics[fileName], uri: "file://"+URI.file(fileName).path});
```

Comment:
I am not sure about this. Will it work for web version? What for should we need add "file://"?

Reply:
For the web version, similar to the Monaco editor, a few additional changes are required. I'll push those by this weekend.